### PR TITLE
Updated env generation for the api file

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

File: scripts/gen-env-files.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ read -p 'Username: ' varKeycloak
 echo 'Enter a username for the API database.'
 read -p 'Username: ' varApiDb
 
-passvar=$(grep -Po 'DB_PASSWORD=\K.*$' ./database/mssql/.env)
+passvar=$(grep -Po '^DB_PASSWORD=\K.*$' ./database/mssql/.env)
```
Comment (FuriousLlama) [inline scripts/gen-env-files.sh:12]: Looks for DB_PASSWORD at the start of the line. Prevents issues where there where multiple passwords commented out

File: scripts/gen-env-files.sh [hunk 2]
```diff
@@ -72,11 +72,28 @@ else
...
 DB_PASSWORD=$passvar
+
 Keycloak__Secret=
-Keycloak__ServiceAccount__Secret=" >> ./backend/api/.env
+Keycloak__ServiceAccount__Secret=
+
+Ltsa__MyLtsaUsername=
+Ltsa__MyLtsaUserPassword=
+Ltsa__IntegratorUsername=
+Ltsa__IntegratorPassword=
```
Comment (FuriousLlama) [inline scripts/gen-env-files.sh:87]: @devinleighsmith These ones were not there but looked necessary given we are retrieving LTSA data. Any concerns?

**Review by Maheshbabu28 — APPROVED** [review]

File: scripts/gen-env-files.sh [hunk 3]
```diff
@@ -72,11 +72,28 @@ else
...
+
+Mayan__BaseUri=http://localhost:7080/api/v4
+Mayan__ConnectionUser=admin
+Mayan__ConnectionPassword=
+
+Cdogs__AuthEndpoint=
+Cdogs__CDogsHost=
+Cdogs__ServiceClientId=
+Cdogs__ServiceClientSecret=" >> ./backend/api/.env
```
Comment (FuriousLlama) [inline scripts/gen-env-files.sh:96]: Followed a similar pattern as the keycloak secret account variable. I think adding those to the user input are more cumbersome than just updating the env file

File: scripts/gen-env-files.sh (hunk 1 above)
Comment (devinleighsmith) [reply scripts/gen-env-files.sh:12]: This makes sense to me.

File: scripts/gen-env-files.sh (hunk 2 above)
Comment (devinleighsmith) [reply scripts/gen-env-files.sh:87]: Having them here is a good idea, as long as none of them are set.

**Review by asanchezr — APPROVED** [review]

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**